### ISSUE TITLE
Download failure for util-macros

### ISSUE THREAD

**Issue by verdurin** [easybuilders/easybuild-easyconfigs#9481]
I see this error when trying to download the sources of `util-macros`:
```
== 2019-12-11 13:50:21,655 easyblock.py:742 DEBUG Trying to download file util-macros-1.19.2.tar.gz from http://cgit.freedesktop.org/xorg/util/macros/snapshot/util-macros-1.19.2.tar.gz to /apps/eb/epyc/sources/x/xorg-macros/util-macros-1.19.2.tar.gz ...
== 2019-12-11 13:50:21,655 filetools.py:509 DEBUG Trying to download util-macros-1.19.2.tar.gz from http://cgit.freedesktop.org/xorg/util/macros/snapshot/util-macros-1.19.2.tar.gz to /apps/eb/epyc/sources/x/xorg-macros/util-macros-1.19.2.tar.gz
== 2019-12-11 13:50:21,655 filetools.py:516 DEBUG Using timeout of 10 seconds for initiating download
== 2019-12-11 13:50:21,655 filetools.py:1292 DEBUG Not creating existing path /apps/eb/epyc/sources/x/xorg-macros
== 2019-12-11 13:50:23,382 filetools.py:559 WARNING URL http://cgit.freedesktop.org/xorg/util/macros/snapshot/util-macros-1.19.2.tar.gz was not found (HTTP response code 400), not trying again
== 2019-12-11 13:50:23,383 filetools.py:585 WARNING Download of http://cgit.freedesktop.org/xorg/util/macros/snapshot/util-macros-1.19.2.tar.gz to /apps/eb/epyc/sources/x/xorg-macros/util-macros-1.19.2.tar.gz failed, done trying
```

and if I try to open the link in a browser:
```
Unsupported snapshot format: util-macros-1.19.2.tar.gz
```

**Comment by boegel** [comment]
This is being fixed in #9477

**Comment by Flamefire** [comment]
This also affects the X11 ECs where xorg-macros is used as a component (and probably not through the xorg-macros EC)

**Comment by boegel** [comment]
`xorg-macros` easyconfigs fixed in #9578

**Comment by Flamefire** [comment]
This can be closed then?

**Comment by boegel** [comment]
X11 easyconfigs also fixed in #9546, so closing this issue...

**Comment by smhr** [comment]
Hi,
The package again is not available. This might work:

**Comment by smhr** [comment]
Hi,
The link to the this package is broken. This might work:
[ftp://ftp.x.org/pub/individual/util/util-macros-1.19.2.tar.bz2](ftp://ftp.x.org/pub/individual/util/util-macros-1.19.2.tar.bz2)

**Comment by boegel** [comment]
@smhr See the fixes in #9578
